stream-chat-android-client/src/main/java/io/getstream/chat/android/client/events/ChatEvent.kt

```diff
@@ -442,7 +448,8 @@ public data class NotificationMarkUnreadEvent(
     val unreadMessages: Int,
     val firstUnreadMessageId: String,
     val lastReadMessageAt: Date,
-    val lastReadMessageId: String,
+    val lastReadMessageId: String?,
```

Comment:
Why is it optional now?
It doesn't make sense for this event.
Could you check with backend?